tests/utils/tools.php

```diff
@@ -632,31 +647,12 @@ function configureFailPoint(MongoDB\Driver\Manager $manager, $failPoint, $mode,
 
     $cmd = new MongoDB\Driver\Command($doc);
     $result = $manager->executeCommand("admin", $cmd);
-    $arr = current($result->toArray());
-    if (empty($arr->ok)) {
-        var_dump($result);
-        throw new RuntimeException("Failpoint failed");
-    }
-    return true;
 }
 
 function failMaxTimeMS(MongoDB\Driver\Manager $manager) {
     return configureFailPoint($manager, "maxTimeAlwaysTimeOut", array("times" => 1));
 }
 
-function getMOUri() {
```

Comment:
Removing this function likely breaks `scripts/start-servers.php` for the Vagrant environment, which was relying on `tools.php`. I realized that while looking at places that still called `getMOPresetBase()`, as I was going to suggest you remove that function as well.

I was going to suggest leaving this here and returning `MONGO_ORCHESTRATION_URI`, but `start-servers.php` doesn't ever include `basic.inc`, which sets that constant. Perhaps we should leave this function in place in this PR and simply create a new ticket to delete both of them later _and_ refactor `start-servers.php`.

Reply:
I have soft-deprecated "start-servers.php", and don't want to spend any time on this. There is already a ticket for getting rid of the Vagrant stuff: https://jira.mongodb.org/browse/PHPC-1242 — I've made a note there to remove this function too.